backend/linux_vt.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Twin - A Tiny Window System
+ * Copyright (c) 2024 National Cheng Kung University, Taiwan
+ * All rights reserved.
+ */
+
+#ifndef _LINUX_VT_H__
+#define _LINUX_VT_H__
+
+#include <fcntl.h>
+#include <linux/kd.h>
+#include <linux/vt.h>
+#include <stdlib.h>
+#include <sys/ioctl.h>
+#include "twin_private.h"
+
+static inline int twin_vt_open(int vt_num)
+{
+    int fd;
+
+    char vt_dev[30] = {0};
+    snprintf(vt_dev, 30, "/dev/tty%d", vt_num);
```

Comment:
Why 30?

Reply:
Initially, we used 30 as a conservative size. Using the command ```find /dev/ -name "tty*"```, we get results like ```/dev/ttyXX```. Therefore, 11 is also a suitable size.